components/combobox/combobox.jsx

```diff
@@ -189,6 +189,7 @@ const propTypes = {
 	multiple: PropTypes.bool,
 	/**
 	 *  Offset adds pixels to the absolutely positioned dropdown menu in the format: ([vertical]px [horizontal]px).
+	 *  This is an undocumented prop and its usage is discouraged.
```

Comment:
Please _completely_ remove `propType` and place this comment in the source code above the line passing it into Dialog component.